package.json

```diff
@@ -12,7 +12,8 @@
     "build": "rimraf lib && tsc -p tsconfig.json",
     "start": "node ./bin/probot run",
     "lint": "tslint --project test",
-    "test": "tsc --noEmit -p test && jest --coverage && npm run lint && npm run doc-lint",
+    "test:watch": "jest --watch --notify",
+    "test": "jest --coverage && npm run lint && npm run doc-lint",
```

Comment:
We don't need to compile first, since `ts-jest` works fine with `jest`

Reply:
I can't find where we added this, but it was because `ts-jest` wasn't actually checking that the typescript successfully compiles. It transpiles it to JS, but is not as strict as the ts compiler, which means the tests could pass, but `npm build` will actually fail.

We should verify this behavior before removing the call to `tsc`.

Reply:
Hmm interesting. I actually got a compilation error on one import with `ts-jest` and not with `tsc`.

I can put it back, just to be sure that everything compiles correctly, because now I'm not sure how to check that it's ok to remove it 😄